### PR TITLE
[WIP] [git] Add links to files from documentation page

### PR DESCRIPTION
**Pull request by alpgarcia**
This PR tries to provide a solution for #442 and #392 

**Needs https://github.com/chaoss/grimoirelab-sirmordred/pull/485 because fixes a dashboard file name that in other case would cause sirmordred to fail when uploading lifecycle dashboard.**

By the time being is just an example using the Git dashboard. Once we agree this is the way to go, I'll modify the rest of the pages to include the same information.

Links to dashboard and index pattern files are added to the
Git dashboard documentation page, so users don't need to figure
out where to find them.

**Note about the icons**: here I share the source of the icons I used to let you check the corresponding licenses:
* [Dashboard icon](https://iconify.design/icon-sets/carbon/dashboard.html): Apache 2.0
* [Index Pattern icon](https://iconify.design/icon-sets/bi/file-ruled.html): MIT

### CODE REVIEW

File: docs/_panels/github-backlog.md [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-This panel focuses on pending tasks, that is, those Issues
-that remain open (note that here we work with data retrieved in the moment of
-dashboard creation).
-
-It is possible to filter data in the same way as in other GitHub panels
-(by submitter, repository and domain).
-
-A list with the oldest Issues is also shown, providing direct
-links to them and some basic information as their titles, how long they have
-been open and when they were created.
```
Comment (mafesan) [inline docs/_panels/github-backlog.md:20]: I checked that this file was renamed to ` docs/_panels/github-issues-backlog.md`. New version of the file [here](https://github.com/chaoss/grimoirelab-sigils/pull/457/files#diff-4e4408d4e8c4b339c4d7de584b2dbab7).

File: docs/_panels/organization-tracking-overview.md [hunk 2]
```diff
@@ -34,4 +34,30 @@ This dashboard is built on top of:
...
+kidash -e https://user:pass@localhost:443/data --import all_enriched-index-pattern.json
+kidash -e https://user:pass@localhost:443/data --import organization_tracking_overview.json
+```
+
+[sirmordred-general]: https://github.com/chaoss/grimoirelab-sirmordred#general
+[dash-icon]: ../assets/images/icons/dashboard.png
+[ip-icon]: ../assets/images/icons/file-ruled.png
+[dashboard]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/json/organization_tracking_overview.json
+[index-pattern]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/json/all_enriched-index-pattern.json
+
```
Comment (mafesan) [inline docs/_panels/organization-tracking-overview.md:62]: I think this blank line should be removed.

File: docs/_panels/github-repositories.md [hunk 3]
```diff
@@ -50,3 +50,32 @@ a different type of items (events) where these values will be available.
...
+```
+
+[github_repositories-schema]: https://github.com/chaoss/grimoirelab-elk/blob/master/schema/github_repos.csv
+[sirmordred-github_repositories]: https://github.com/chaoss/grimoirelab-sirmordred#repo
+[dash-icon]: ../assets/images/icons/dashboard.png
+[ip-icon]: ../assets/images/icons/file-ruled.png
+[index-pattern]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/json/github_repositories-index-pattern.json
+[dashboard]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/json/github_repositories.json
+
+
```
Comment (mafesan) [inline docs/_panels/github-repositories.md:81]: I think this blank line should be removed (This is not important, but I observed most of the modified .md files have 0 or 1 blank lines at the end).

File: docs/_panels/github-pull-requests-comments-and-collaboration.md [hunk 4]
```diff
@@ -72,4 +72,31 @@ The fields that can be used to aggregate data from both issues and pull requests
...
+```
+
+[github2_pull_requests-schema]: https://github.com/chaoss/grimoirelab-elk/blob/master/schema/github2_pull_requests.csv
+[sirmordred-github2_pull_requests]: https://github.com/chaoss/grimoirelab-sirmordred#github2-
+[dash-icon]: ../assets/images/icons/dashboard.png
+[ip-icon]: ../assets/images/icons/file-ruled.png
+[index-pattern]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/json/github2_pull_requests-index-pattern.json
+[dashboard]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/json/github2_pull_requests_comments_and_collaboration.json
+
+
```
Comment (mafesan) [inline docs/_panels/github-pull-requests-comments-and-collaboration.md:102]: I think this blank line should be removed (This is not important, but I observed most of the modified .md files have 0 or 1 blank lines at the end).

File: docs/_chaoss-gmd-cde/maintainer_merge_request_duration.md [hunk 5]
```diff
@@ -31,3 +31,30 @@ widget.
...
+kidash -e https://user:pass@localhost:443/data --import github_issues-index-pattern.json
+kidash -e https://user:pass@localhost:443/data --import maintainer_response_to_merge_request_duration.json
+```
+
+[github_issues-schema]: https://github.com/chaoss/grimoirelab-elk/blob/master/schema/github_issues.csv
+[sirmordred-github_issues]: https://github.com/chaoss/grimoirelab-sirmordred#github-
+[dash-icon]: ../assets/images/icons/dashboard.png
+[ip-icon]: ../assets/images/icons/file-ruled.png
+[index-pattern]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/chaoss-gmd-cde/github_issues-index-pattern.json
+[dashboard]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/chaoss-gmd-cde/maintainer_response_to_merge_request_duration.json
```
[the remark this answers is not in the document]
Comment (alpgarcia) [reply docs/_chaoss-gmd-cde/maintainer_merge_request_duration.md:60]: I guess this shouldn't be done as part of this PR as it has nothing to do with the documentation. I would open an issue or a PR proposing the change instead.

File: docs/_panels/github-backlog.md (hunk 1 above)
Comment (alpgarcia) [reply docs/_panels/github-backlog.md:20]: Yes, this was outdated. We had documented the old dashboard that was split into two.

File: docs/_panels/data-status.md [hunk 6]
```diff
@@ -1,39 +1,67 @@
...
+
+## Field Description
+
+Brief description of what used fields should contain:
+
+- `metadata__updated_on`: date of creation or last update of an item in its data source (git, gerrit, etc.)
+- `metadata__timestamp`: date in which a given item was indexed in Elastic Search.
+
+Example:
+An item that was created in git on 2014-10-21 and it is recovered and indexed today in our system will have 2014-10-21 stored in `metadata__updated_on` and today's date in `metadata__timestamp`. Here we show simplified dates, as you can see in Data Status Panel we are storing dates with milliseconds precision.
```
[the remark this answers is not in the document]
Comment (alpgarcia) [reply docs/_panels/data-status.md:39]: That's probably due to CR LF characters due to cross-platform edition:
https://docs.github.com/en/github/using-git/configuring-git-to-handle-line-endings

File: docs/_panels/github-pull-requests-comments-and-collaboration.md (hunk 4 above)
Comment (alpgarcia) [reply docs/_panels/github-pull-requests-comments-and-collaboration.md]: Now each and every file should have a single new line at the end.

**Comment by alpgarcia** [comment]
@mafesan please review my comments and modifications. Thanks!

File: docs/_chaoss-gmd-cde/maintainer_merge_request_duration.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (mafesan) [reply docs/_chaoss-gmd-cde/maintainer_merge_request_duration.md:60]: You are totally right. As a reference, I'm posting a link to the related issue: https://github.com/chaoss/grimoirelab-sigils/issues/468.

**Pull request merged**